code/modules/clothing/head/misc_special.dm

```diff
@@ -130,10 +130,10 @@
 	..()
 
 	if(!isturf(user.loc))
-		to_chat(user, "You cannot turn the light on while in [user.loc]") //To prevent some lighting anomalities.
-		return
+		to_chat(user, SPAN_WARNING("You cannot turn the light [on ? "off" : "on" ] while in [user.loc].")) //To prevent some lighting anomalies.
+		return FALSE
```

Comment:
Why was this changed from return to return FALSE?

Reply:
removed